pom.xml

```diff
@@ -488,7 +488,7 @@
         <version.org.hibernate.validator>8.0.0.Final</version.org.hibernate.validator>
         <version.org.hornetq>2.4.9.Final</version.org.hornetq>
         <version.org.infinispan>13.0.11.Final</version.org.infinispan>
-        <version.org.infinispan.protostream>4.4.3.Final</version.org.infinispan.protostream>
+        <version.org.infinispan.protostream>4.4.4.Final</version.org.infinispan.protostream>
```

Comment:
FWIW I didn't initally bump the version since this Infinispan version is still using 4.4.3.Final – https://github.com/infinispan/infinispan/blob/13.0.11.Final/build-configuration/pom.xml#L186

Reply:
Looks like the only code changes are these 2 lines - https://github.com/infinispan/protostream/compare/4.4.3.Final...4.4.4.Final#diff-8749a97b010e848895a578777761f9da9a082e99685b3f1b8232837228f61361R271-R272

Reply:
Fixes https://issues.redhat.com/browse/IPROTO-218. Good then.